### PR TITLE
Added support for Android Appium/e2e tests

### PR DESCRIPTION
**Pull request by vitalyiegorov**

### CODE REVIEW

**Comment by vitalyiegorov** [comment]
According to this article as of `react-native: "^0.64.0"` this is not needed anymore https://dev.to/nextlevelbeard/an-end-to-the-abuse-of-accessibility-ids-5d2j

File: src/index.tsx [hunk 1]
```diff
@@ -270,7 +270,9 @@ const OtpInputs = forwardRef<OtpInputsRef, Props>(
             ref={inputs.current[inputIndex]}
             secureTextEntry={secureTextEntry}
             selectTextOnFocus={selectTextOnFocus}
+            accessible={true}
```
Comment (dsznajder) [inline src/index.tsx:273]: Just `accessible` is enough I think :)
```suggestion
            accessible
```

**Review by dsznajder — APPROVED** [review]
❤️

File: src/index.tsx (hunk 1 above)
Comment (vitalyiegorov) [reply src/index.tsx:273]: Tried, unfortunately - no, current React Native Android implementation does not correctly utilises resource-id generation for TextInput, and only addition of `accessibilityLabel` will create `content-desc` attribute in native elements which can be found using `$('~testID')` locator strategy via Appium

Comment (vitalyiegorov) [reply src/index.tsx:273]: Used to strict typescript and eslint checks, this should also work

**Pull request merged**